tests/unittest/_torch/modules/mamba/benchmark_replay_selective_state_update.py

```
@@ -0,0 +1,807 @@
+                    use_internal_pdl=args.internal_pdl,
+                    _block_size_m=block_size_m,
+                    _num_warps=num_warps,
+                    _num_stages=num_stages,
+                    _precompute_num_warps=precompute_num_warps,
+                    _precompute_num_stages=precompute_num_stages,
+                    _heads_per_block=heads_per_block,
+                    **extra_kwargs,
+                )
+
+            parts = []
+            if block_size_m is not None: parts.append(f"M={block_size_m}")
```

Flake8 (7.3.0)

[error] multiple statements on one line (colon)

(E701)